src/database/migrations/20190704115631-create-users.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Articles', {
```

Comment:
The title of this migrations reads ```create-users```, is there any reason why it creates Articles when read?